grayskull/base/base_recipe.py

```diff
@@ -178,9 +178,26 @@ def generate_recipe(
 
         :param folder_path: Path to the folder
         """
-        recipe_dir = Path(folder_path) / self.get_var_content(
+        print(f"This is the folder_path: {folder_path}")
```

Comment:
Let's get rid of the print statements (all of them!) before the merge

Reply:
Done!